src/test/resources/com/puppycrawl/tools/checkstyle/checks/imports/unusedimports/InputUnusedImports3.java

```diff
@@ -0,0 +1,44 @@
+/*
+UnusedImports
+processJavadoc = (default)true
+
+
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.imports.unusedimports;
+
+import java.awt.AWTException;
+import java.awt.Rectangle; // violation 'Unused import - java.awt.Rectangle'
+import java.awt.event.InputEvent;
+import java.awt.event.KeyEvent; // violation 'Unused import - java.awt.event.KeyEvent'
```

Comment:
Unrelated to pitest:
Is this a defect ?
This import is used in javadoc, it should not be violation.

If you agree please recheck our issue tracker for such issue, if not exist, please open new issue on us.
And we will put here comment `until #....`

Reply:
As per the mutation if I remove `.filter(JavadocTag::canReferenceImports)` the violation will be removed from line 13
Which I don't expect.

As per my understanding of check we only supprot the key word in javadoc if it is refrenced as `{@link KeyEvent}` then it is violation but in this case as what i understand from https://checkstyle.org/config_imports.html#UnusedImports


in java doc it is written as simple text 
```
     * @param keycode which key to press. For example, KeyEvent

```
then it is not violation if the same thing is like 
`     * @param keycode which key to press. For example, {@link KeyEvent}` 
Then it is issue